sx-question-print.el

```diff
@@ -179,7 +179,12 @@ QUESTION must be a data structure returned by `json-read'."
   ;; Print everything
   (sx-question-mode--print-section question)
   (sx-assoc-let question
-    (mapc #'sx-question-mode--print-section .answers))
+    (mapc #'sx-question-mode--print-section
+          (cl-sort .answers
```

Comment:
I looked at the docs for `sort` (well, the `eldoc`) and it said the sequence had to be a list. I made this patch very on-the-fly this morning and I was running late for work, so I didn't try an alternative XD

Does `sort` work for vectors? That's what `.answers` is.